lib/types/types.ts

```diff
@@ -0,0 +1,74 @@
+export interface UserData {
+  id: number;
+  userId: string;
+  name: string;
+  lastname: string;
+  email: string;
+  username: string;
+  bridgeUser: string;
+  password: Password;
+  mnemonic: string;
+  root_folder_id: number;
+  hKey: HKey;
+  secret_2FA: any;
+  errorLoginCount: number;
+  is_email_activity_sended: boolean;
+  referralCode: string;
+  referrer: any;
+  syncDate: any;
+  uuid: string;
+  lastResend: any;
+  credit: number;
+  welcomePack: boolean;
+  registerCompleted: boolean;
+  backupsBucket: any;
+  sharedWorkspace: boolean;
+  tempKey: any;
+  avatar: any;
+  emailVerified: boolean;
+  createdAt: string;
+  updatedAt: string;
+}
+
+interface Password {
+  type: string;
+  data: number[];
+}
+
+interface HKey {
+  type: string;
+  data: number[];
+}
+
+export interface CheckIfUserHasSubscription {
```

Comment:
I suggest you to name this `CheckIfUserHasSubscriptionResponse`. Simply because `Response` associates this with an HTTP request tipically, so you don't need to think what is this type for